### PR TITLE
[IMP] account: Add a menu Product under the configuration.

### PR DESCRIPTION
**Pull request by kme-odoo**
Task : https://www.odoo.com/web?#id=28342&view_type=form&model=project.task&action=327&menu_id=4720
Pad : https://pad.odoo.com/p/r.c2340efda0a1f4c110d4cb3c8418d29a

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:
## 

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/views/account_view.xml [hunk 1]
```diff
@@ -1905,6 +1905,34 @@
...
+        <record id="product_product_view_tree" model="ir.ui.view">
+            <field name="name">product.product.tree</field>
+            <field name="model">product.product</field>
+            <field name="arch" type="xml">
+                <tree string="Products">
+                    <field name="default_code"/>
+                    <field name="name"/>
+                    <field name="attribute_value_ids" widget="many2many_tags" groups="product.group_product_variant"/>
+                    <field name="lst_price"/>
+                    <field name="taxes_id"/>
```
Comment (mdi-odoo) [inline addons/account/views/account_view.xml]: Add widget="many2many_tags". Currently it shows **(0 records)**, if there is no records.

File: addons/account/views/account_view.xml [hunk 2]
```diff
@@ -1905,6 +1905,34 @@
...
+                    <field name="supplier_taxes_id"/>
```
Comment (mdi-odoo) [inline addons/account/views/account_view.xml]: Same here .

File: addons/account/views/account_view.xml [hunk 3]
```diff
@@ -1905,6 +1905,34 @@
...
+        <record id="product_product_action" model="ir.actions.act_window">
+            <field name="name">Products</field>
+            <field name="type">ir.actions.act_window</field>
+            <field name="res_model">product.product</field>
+            <field name="view_mode">kanban,tree,form</field>
+            <field name="view_type">form</field>
+            <field name="view_id" ref="product_product_view_tree"/>
+        </record>
+
+        <menuitem id="menu_product_product_action" name="Product"
```
Comment (mdi-odoo) [inline addons/account/views/account_view.xml]: Menu name should be Product**s**.

**Comment by kme-odoo** [comment]
Work Continue at https://github.com/odoo/odoo/pull/13490

**Pull request closed without merging**